packages/signals/browser-signals/src/types/signals.ts

```diff
@@ -45,10 +49,28 @@ export type InstrumentationSignal = AppSignal<
   InstrumentationData
 >
 
+type NetworkRequestData = {
+  action: 'Request'
+  url: RequestInfo | URL
```

Comment:
Since this is going to be serialized both in the sandbox and when sent to the server, we should only have JSON types in our signal API. i.e this should be a string.